### DISCUSSION TITLE
Can `ElectroError` be exported as a class please?

### DISCUSSION THREAD

**Discussion by arpadgabor** [tywalch/electrodb#157] [Ideas]
At the moment the `ElectroError` is exported as an interface. Most libraries export these errors so we can easily check if an error is of theirs with `error instanceOf ElectroError` and we are also getting nice type safety this way. I see the error is already declared as a class inside JS but the type definitions export it as an interface.

**Comment by tywalch** [comment]
Great idea 👍

**Comment by tywalch** [comment]
@arpadgabor Error classes are now exported in `2.1.2`

**Reply by arpadgabor** [reply]
Works great! Thanks a lot!

**Reply by Easy-Cloud-in** [reply]
Hi, Can you show me how to catch electrodb errors?
